bigtable/admin/admin_client.cc

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/admin/admin_client.h"
+
+#include <absl/memory/memory.h>
+
+namespace {
+/// An implementation of the bigtable::AdminClient interface.
```

Comment:
This is basically an English version of the next line; maybe rewrite it in terms of what it can vs. cannot do, instead of the mechanics of how it did this?

Reply:
Done.